### DISCUSSION TITLE
Welcome to Volume2 Discussions!

### DISCUSSION THREAD

**Discussion by irzyxa** [irzyxa/Volume2#242] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jpsala** [comment]
Hello @irzyxa, hello guys,
I'm Juan Pablo Sala, a developer, I have some apps that I install on every box, like autohotkey, shareX, copyQ, strokesplus.net, and always ```Volume2```, I love this program, I was always struggling to get a good tray volume app and never found one, until now.
Quick question, in the portable version, where are the settings stored?

**Reply by irzyxa** [reply]
For portable version you should put Config.ini to the root folder. I do not provide default ini because you can easily replace yours when update and lose your settings. So here is the default Config.ini
[Config.zip](https://github.com/irzyxa/Volume2/files/11192522/Config.zip)

**Reply by jpsala** [reply]
@irzyxa, I have my volume2 modified, I mean, I made some changes in the settings, but there is no Config.zip in my root folder, is it ok?

**Reply by irzyxa** [reply]
I attached zip because it is not possible to attach ini file. You can open settings, System tab and click Open profile folder to see where is your Config.ini

**Reply by jpsala** [reply]
Sorry, I meant config.ini, there is no config.ini :) where are the settings I already modified if there is no config.ini, sorry to ask

**Reply by irzyxa** [reply]
So it can be in the root folder or in the profile folder. The root folder is preferred
